beamer/models/claim.py

```diff
@@ -39,7 +39,11 @@ def __init__(
         | challenger_winning.to(claimer_winning)
         | ignored.to(ignored)
     )
-    invalidate = claimer_winning.to(invalidated) | challenger_winning.to(invalidated)
+    invalidate = (
+        claimer_winning.to(invalidated)
+        | challenger_winning.to(invalidated)
+        | invalidated.to(invalidated)
```

Comment:
when is this transition triggered?

Reply:
For example when we trigger the invalidation and then receive the event later.